server/helpers/funtions.js

```diff
@@ -5,3 +5,14 @@ export const getMoneyFormat = (number) =>
   new Intl.NumberFormat('en-NG', { style: 'currency', currency: 'NGN' }).format(number);
 
 export const getFormattedName = (name) => `"${name}"`;
+
+export const slugifyString = (string) =>
```

Comment:
Not sure why String was added to the name of the helper class and used as the parameter name. We can also slugify numbers too 😼 

```suggestion
export const slugify = (text) =>
```